plaidml/bridge/openvino/ops/prior_box.cpp

```diff
@@ -0,0 +1,440 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset4.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;  // NOLINT[build/namespaces]
+using ngraph::opset4::PriorBox;
+
+namespace PlaidMLPlugin {
+
+struct LocalConfigs {
+  int H;
+  int W;
+  int IH;
+  int IW;
+  int num_of_prior;
+  int min_element_size;
+  float step;
+  float step_x;
+  float step_y;
+  float offset;
+  std::vector<float> aspect_ratios;
+  std::vector<float> aspect_ratios_scale_size;
+  std::vector<float> variance;
+  std::vector<float> min_size;
+};
+
+// Preprocess data with some attributes
+void prepareConfig(LocalConfigs& conf, const ngraph::op::PriorBoxAttrs& attrs, const Context& ctx) {
```

Comment:
This kind of seems like the constructor for `LocalConfigs`. I suggest we rename `LocalConfigs` to `PriorBoxImpl`, and then add methods to it and do most of the work in this type.

Reply:
Move all to PriorBoxImpl class now.